tests/suites/utils.js

```diff
@@ -309,12 +309,12 @@ casper.test.begin('isWebPage() tests', 3, function(test) {
     test.done();
 });
 
-casper.test.begin('isJsFile() tests', 5, function(test) {
+casper.test.begin('isJsFile() tests', 4, function(test) {
     var testCases = {
         '':             false,
         'toto.png':     false,
         'plop':         false,
-        'gniii.coffee': '.coffee' in require.extensions,
+//        'gniii.coffee': '.coffee' in require.extensions,
```

Comment:
Hmm but before your pull request, this test just works as expected right ?
Also, we are still compatible with phantomjs 1.9.8 and so on we need to excecute coffee tests in this case.